front-end/src/Components/Layouts/Organization.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Button,
+  Card,
+  Container,
+  Grid,
+  Typography,
+  ListItem,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+  Fab,
+  makeStyles,
+  Paper,
+} from "@material-ui/core";
+import React, { Fragment } from "react";
+import { NotificationMessage } from "../Misc/Notifications";
+import EditIcon from "@material-ui/icons/Edit";
+import EventIcon from "@material-ui/icons/Event";
+import DeleteIcon from "@material-ui/icons/Delete";
+interface EventInfoProps {
+  organizationData: any;
+  key: string;
+  setEventOrgId: Function;
+  setEditOrganizationData: Function;
+  setNotification: (notification: NotificationMessage) => void;
+  classes: any;
+}
+
+const useStyles = makeStyles({
```

Comment:
This is now unused. Can we remove it to keep things clean?

Reply:
Done